README.md

```diff
@@ -40,7 +40,14 @@ sudo snap install --edge nvim --classic
 
 [Other methods of installation are available in the official repository of Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim)
 
-Instructions for **NodeJS** are also available in [the official repo.](https://github.com/nodesource/distributions/blob/master/README.md)
+
+To install **NodeJS** in the most convinient way, install NodeJS via NVM (Node Version Manager).
```

Comment:
We are not talking about the most suitable for this purpose (convenient) way of Node JS installation

Reply:
I have noted two ways to install NodeJS. One of them should be mentioned as more preferable.

Reply:
Please, check the meaning of word convenient word and the usage of it your sentence